### PR TITLE
Add adapter for the kitty terminal and config docs to Readme.

### LINKED ISSUE #207: Kitty support
Is it possible to add or build kitty support? How would I go about that?

### PR DESCRIPTION
**Pull request by cnschn**
Closes #207. This is still pending some clarification on kitty's file type support for background images, see [this discussion](https://github.com/kovidgoyal/kitty/discussions/7636) - at least for me this PR just works, even though it really shouldn't?

### CODE REVIEW

**Comment by cnschn** [comment]
Update: Kitty does support other image formats, the docs have been updated to clarify this. I'll check which version of kitty introduced the support to note that in the Readme, then we should be good to go.

**Comment by sylveon** [comment]
Awesome!

**Comment by cnschn** [comment]
This was quite the rabbit hole (currently the actual implementation for some of the kitty commands are compiled go code, which is automatically generated from python during the build!), but I figured out that the support for other image formats was there since `kitty==0.27.0`. That version was released on 2023-01-31, but apparently a few distros still package older versions, so I added the minimum version to the README as well.

File: pokemonterminal/terminal/adapters/kitty.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class KittyProvider(_TProv):
+    def is_compatible() -> bool:
+        return "KITTY_WINDOW_ID" in os.environ
+
+    def change_terminal(path: str):
+        try:
+            run(["kitty", "@", "set-background-image", path], check=True, capture_output=True)
+        except CalledProcessError as err:
+            print_kitty_error(err)
+            sys.exit(-1)
```
Comment (sylveon) [inline pokemonterminal/terminal/adapters/kitty.py:23]: Other adapters don't call sys.exit on failure, so I would try and avoid it here

Comment (cnschn) [reply pokemonterminal/terminal/adapters/kitty.py]: Ah, sure. Updated.

**Comment by sylveon** [comment]
Thanks for the contribution!

**Pull request merged**